### ISSUE TITLE
Improve accessibility of list items for Talkback screen reader

### ISSUE THREAD

**Issue by aryanchoudharypro** [keymapperorg/KeyMapper#1369]
**Developer TODO (don't remove)**
- [ ] update documentation
hello. i would like the buttens in the app to be labled properly so that talkback can read them. also, there should be custom accessibility actions to drag items in the app, as we are blind and don't know where we have to dropp the volume up key if we are dragging it in the triger list.

**Comment by aryanchoudharypro** [comment]
After beta 3.0, the ui is much better, but the create new key map butten is not labelled

**Comment by sds100** [comment]
Think that should be fixed now ;)

**Comment by aryanchoudharypro** [comment]
Now talkback says new key map two times instead of only speaking it once

**Comment by aryanchoudharypro** [comment]
Also the options in the option tab of a key map do not announce their states of switches, for example,  Allow other apps to trigger this key map should say whether it's off or on

**Comment by aryanchoudharypro** [comment]
Also, TalkBack saying new key map 2 times only happens when no keymaps are in the list. When I have imported some key maps it does not say  it incorrectly and says new key map as it should

**Comment by aryanchoudharypro** [comment]
@claude how much work is it to implement accessibility actions to move triggers up and down?
